docs/internationalisation_guide.md

```diff
@@ -1,38 +1,32 @@
 # Internationalisation guide
 
-This is mostly standard [Rails i18n](http://guides.rubyonrails.org/i18n.html) - Translations are stored in `config/locales/`, with a `.yml` file per locale.
+This is mostly standard [Rails i18n](http://guides.rubyonrails.org/i18n.html):
 
-If translation value is missing from a locale file then the EN value will be used instead.
+- Translations are stored in `config/locales/`, with a `.yml` file per locale.
+- If translation value is missing from a locale file then the EN value will be used instead.
 
 ## Changing an existing translation key
 
-Edit the value of EN locale, you should then _manually_ edit all other locales to set the altered translated value to be blank.
+1. Edit the value of the EN locale
+1. You then need to set the non-EN locale values to be blank. You can do this manually or follow the [rails_translation_manager documentation](https://github.com/alphagov/rails_translation_manager#i18n-tasks)
 
 ## Adding a new translation key
 
-_Manually_ create the key in `en.yml`, with the english text.
-
-Run a task to add that key to all other language files:
-
-```
-bundle exec rake translation:regenerate
-```
+1. Manually create the key in `en.yml`, with the english text.
+1. Then run `rake translation:add_missing` (see [rails_translation_manager documentation](https://github.com/alphagov/rails_translation_manager#rake-command-reference))
 
 ### Pluralised translations
 
-For terms that are translatable in both singular and plural forms (e.g. document.type.publication), include "one" and "other"
-keys for the singular and plural translation of the term.
+For terms that are translatable in both singular and plural forms (e.g. [`document.type.publication`](https://github.com/alphagov/whitehall/blob/bd1dcd37fbf2c91a10c7927d1d46b3df75cdca2d/config/locales/en.yml#L294-L296)), include "one" and "other" keys for the singular and plural translation of the term.
+
+Note that other languages have different pluralisation rules (such as "few" and "many") but these other plural forms should be [left blank](https://github.com/alphagov/whitehall/blob/02666684c7869cc8b633f6382be0ebe6ee22e3b0/config/locales/pl.yml#L401-L405)).
 
-Note: pluralised translation terms should only ever contain these two plural form keys in en.yml, otherwise the code that
-regenerates the other translation locale files will not recognise them as being plural translations and will not generate
-the correct pluralisation keys for the different locales.
+Note: pluralised translation terms should only ever contain these two plural form keys in `en.yml`, otherwise the code that regenerates the other translation locale files will not recognise them as being plural translations and will not generate the correct pluralisation keys for the different locales.
```

Comment:
The `LocaleChecker` will warn if there are unrecognised pluralisation forms (e.g., `:many` for English, or anything where one of the keys is `:other`). I wonder if this actually means it _will_ generate them but that this will be an error for other reasons? Or does it mean that this breaks generation _and_ breaks separately if the plurals are modified manually later?

Reply:
Thanks Ben - I didn't get as far as trying this out locally to see what happens (this paragraph was carried over from what already existed) but agree that documenting the exact consequences is not a blocker.